lib/add-to-org/views/success.erb

```diff
@@ -6,7 +6,7 @@
   </head>
   <body>
     <h1>You're all set</h1>
-    <p class="lead">You simply need to <a href="https://github.com/orgs/government/invitation?return_to=<%= ERB::Util.html_escape(redirect) %>">confirm your invitation to join the organization</a> on GitHub.</p>
+    <p class="lead">You simply need to <a href="https://github.com/orgs/<%= ENV['GITHUB_ORG_ID'] %>/invitation?return_to=<%= ERB::Util.html_escape(redirect) %>">confirm your invitation to join the organization</a> on GitHub.</p>
```

Comment:
Ahh, nice catch. 

The "right" way to do this would be to pass it as a local by adding it to https://github.com/benbalter/add-to-org/blob/master/lib/add-to-org.rb#L61 like so:

```ruby
success({ :redirect => "https://github.com#{path}", :org_id => org_id })
```

That'll pull the org id from [the `org_id` method](https://github.com/benbalter/add-to-org/blob/master/lib/add-to-org/helpers.rb#L33-L35), in case we add some sort of validation down the line, and will make it available within the template as `<%= org_id %>`.